Tan_ShinYi/Assignments/Flask_Fundamentals/6_Dojo_Survey_Validation/templates/index.html

```diff
@@ -0,0 +1,54 @@
+<html>
+    <head>
+       <title>Dojo Survey Index</title>
+       <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='my_style1.css') }}">
+    </head>
+    <body>
+      <div id="getinfo">
+        {% with messages=get_flashed_messages()%}
+          {% if messages %}
+            {% for message in messages %}
+              <p>{{message}}</p>
+            {% endfor %}
+          {% endif %}
+        {% endwith %}
+        <form action='/users' method='post' id='userform'>
+          <table>
+            <tbody>
+              <tr>
+                <td>Your Name:</td>
+                <td><input type='text' name='name' value="{{session['name']}}"></td>
+              </tr>
+              <tr>
+                <td>Dojo Location:</td>
+                <td>
+                  <select name="location">
```

Comment:
TODO: Figure out how to automatically set a specific option in an HTML select, so that you can default it to {{session['location']}} etc